osbenchmark/client.py

```diff
@@ -282,23 +281,35 @@ def parse_aws_log_in_params(self):
 
     def create(self):
         # pylint: disable=import-outside-toplevel
-        import opensearchpy
+        import osbenchmark.sync_connection
         from botocore.credentials import Credentials
 
+        if "connection_class" in self.client_options:
+            connection = self.client_options.pop("connection_class", None)
+            if connection.lower() in {"requestshttpconnection", "requests"}:
+                connection_class = osbenchmark.sync_connection.RequestsHttpConnection
+            else:
+                connection_class = osbenchmark.sync_connection.Urllib3HttpConnection
+        else:
+            connection_class = osbenchmark.sync_connection.Urllib3HttpConnection
+
         if "amazon_aws_log_in" not in self.client_options:
-            return opensearchpy.OpenSearch(hosts=self.hosts, ssl_context=self.ssl_context, **self.client_options)
+            return self.BenchmarkOpenSearch(hosts=self.hosts, ssl_context=self.ssl_context,
```

Comment:
Would suggest renaming this to `BenchmarkSyncOpenSearch` to match the other class.  Likewise with `create()` vis-a-vis `create_async()`.  Please add comments prior to each indicating the presence of the other.

Reply:
Actually, you might. consider renaming both classes to be more descriptive, something like _PythonSyncClient_ and _PythonAsyncClient_, so future clients can be named accordingly.